renderdoc/driver/vulkan/vk_resources.cpp

```diff
@@ -1702,6 +1702,351 @@ uint32_t GetPlaneByteSize(uint32_t Width, uint32_t Height, uint32_t Depth, VkFor
   return GetByteSize(Width, Height, Depth, Format, mip);
 }
 
+struct TexelShape
+{
+  uint32_t width;
+  uint32_t height;
+  uint32_t bytes;
+};
+
+TexelShape GetTexelShape(VkFormat Format, uint32_t plane)
+{
+  switch(Format)
+  {
+    case VK_FORMAT_R64G64B64A64_UINT:
+    case VK_FORMAT_R64G64B64A64_SINT:
+    case VK_FORMAT_R64G64B64A64_SFLOAT: return TexelShape{1, 1, 32};
```

Comment:
I'd prefer not using the new modern C++ constructor syntax with `{}`s. It's fine to just do `return {1, 1, 32}` for tiny POD structs like this, but otherwise I'd rather have a normal constructor with `()s`.

Reply:
Done.